targets/DeployDependencies.proj

```diff
@@ -20,17 +20,40 @@
           SkipUnchangedFiles="true"           
           />
   </Target>
+  
+  <!-- Copy content of BuildTools package to deployment folders -->
+  <Target Name="CopyBuildTools">
+    <PropertyGroup>
+      <BuildToolsVersionFile>$(RepoRoot)BuildToolsVersion.txt</BuildToolsVersionFile>
+    </PropertyGroup>
+    <ReadLinesFromFile File="$(BuildToolsVersionFile)">
+      <Output TaskParameter="Lines" ItemName="BuildToolsVersion" />
+    </ReadLinesFromFile>
+    <PropertyGroup>
+      <BuildToolsVersion>@(BuildToolsVersion)</BuildToolsVersion>
+    </PropertyGroup>
+    <ItemGroup>
+      <BuildToolsFiles Include="$(ToolPackagesDir)Microsoft.DotNet.BuildTools\$(BuildToolsVersion)\lib\*.*" />
```

Comment:
The build tools package copies all the files.  Initially I tried copying a subset in this target, but got errors because files were missing.  So instead of trying to find the right set of files through trial and error I decided to just copy all the files.